corgi/api/views.py

```diff
@@ -380,12 +381,49 @@ class ComponentViewSet(ReadOnlyModelViewSet):  # TODO: TagViewMixin disabled unt
     filterset_class = ComponentFilter
     lookup_url_kwarg = "uuid"
 
+    def get_queryset(self):
+        mode = self.request.query_params.get("mode")
+        if mode == "latest":
```

Comment:
Do we expect any other modes to ever exist? I think this would be fine as `show_latest=true|false` boolean flag, no?

Reply:
yes (in discussion with Marek) but equally happy to constrain to show_latest for now

Reply:
I think we should default to showing only the latest for ofuri param on components, and have a boolean to `show_all`.

Reply:
ok - after chatting with @jasinner  - we will bake behaviour into ?ofuri={} usage and not implement any additional url param switches eg. 'latest' only makes sense in context of a specific offering/product ... and we are always free to facet on ofuri with ?product_streams=ofuri (if we do not want latest mode).

Note - 'latest' should be used with manifest selection as well